src/sentry/static/sentry/app/views/projectDetail/projectLatestAlerts.tsx

```diff
@@ -0,0 +1,245 @@
+
+    // we are listing 3 alerts total, first unresolved and then we fill with resolved
+    return [
+      [
+        'unresolvedAlerts',
+        `/organizations/${organization.slug}/incidents/`,
+        {query: {...query, status: IncidentStatus.OPENED}},
+      ],
+      [
+        'resolvedAlerts',
+        `/organizations/${organization.slug}/incidents/`,
+        {query: {...query, status: IncidentStatus.CLOSED}},
+      ],
+    ];
+  }
+
+  /**
+   * If our alerts are empty, determine if we've configured alert rules (empty message differs then)
+   */
+  async onLoadAllEndpointsSuccess() {
+    const {unresolvedAlerts, resolvedAlerts} = this.state;
+    const {location, organization} = this.props;
+
+    if ([...(unresolvedAlerts ?? []), ...(resolvedAlerts ?? [])].length !== 0) {
+      this.setState({hasAlertRule: true});
+      return;
+    }
+
+    this.setState({loading: true});
+
+    const alertRules = await this.api.requestPromise(
+      `/organizations/${organization.slug}/alert-rules/`,
+      {
+        method: 'GET',
+        query: {
+          ...pick(location.query, [...Object.values(URL_PARAM)]),
+          per_page: 1,
+        },
+      }
+    );
+
+    this.setState({hasAlertRule: alertRules.length > 0, loading: false});
+  }
+
+  renderAlertRow = (alert: Incident) => {
+    const {organization, theme} = this.props;
+    const isResolved = alert.status === IncidentStatus.CLOSED;
+    const isWarning = alert.status === IncidentStatus.WARNING;
+
+    const color = isResolved ? theme.gray200 : isWarning ? theme.yellow300 : theme.red300;
+    const Icon = isResolved ? IconCheckmark : isWarning ? IconWarning : IconFire;
+
+    return (
+      <AlertRowLink
+        to={`/organizations/${organization.slug}/alerts/${alert.identifier}/`}
+        key={alert.id}
+      >
+        <AlertBadge color={color} icon={Icon}>
+          <AlertIconWrapper>
+            <Icon color="white" />
+          </AlertIconWrapper>
+        </AlertBadge>
+        <AlertDetails>
+          <AlertTitle>{alert.title}</AlertTitle>
+          <AlertDate color={color}>
+            {isResolved
+              ? tct('Resolved [date]', {date: <TimeSince date={alert.dateClosed!} />})
+              : tct('Triggered [date]', {date: <TimeSince date={alert.dateStarted} />})}
+          </AlertDate>
+        </AlertDetails>
+      </AlertRowLink>
+    );
+  };
+
+  renderInnerBody() {
+    const {organization, projectSlug} = this.props;
+    const {loading, unresolvedAlerts, resolvedAlerts, hasAlertRule} = this.state;
+    const alertsUnresolvedAndResolved = [
+      ...(unresolvedAlerts ?? []),
+      ...(resolvedAlerts ?? []),
+    ];
+    const checkingForAlertRules =
+      alertsUnresolvedAndResolved.length === 0 && hasAlertRule === undefined;
+    const showLoadingIndicator = loading || checkingForAlertRules;
+
+    if (showLoadingIndicator) {
+      return <Placeholder height="160px" />;
+    }
+
+    if (!hasAlertRule) {
+      return (
+        <div>
```

Comment:
do you need this `div` here?

Reply:
Will check both of these in a follow-up.